### PR TITLE
Improve checking instance status

### PR DESCRIPTION
**Pull request by dokshina**
Add `cartridge status` command
Improve `start/stop` logs using instance status

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,6 +112,7 @@ These commands are supported:
 * `build` - build the application for local development and testing;
 * `start` - start a Tarantool instance(s);
 * `stop` - stop a Tarantool instance(s);
+* `status` - get current instance(s) status;
```
Comment (lenkis) [inline README.md:115]:
```suggestion
* `status` - get instance(s) current status;
```

File: README.md [hunk 2]
```diff
@@ -356,12 +357,19 @@ all defined instances:
 # in the application directory
 cartridge start # starts all instances
 cartridge start .router_1 # start single instance
+cartridge start .router_1 --stateboard # start single instance and stateboard
```
Comment (lenkis) [inline README.md:360]:
```suggestion
cartridge start .router_1 --stateboard # start a single instance and stateboard
```

File: README.md [hunk 3]
```diff
@@ -376,6 +384,24 @@ These options from the `start` command are supported:
 * `--stateboard`
 * `--stateboard-only`
 
+<!--
+Please, update cmd_status usage in cartridge-cli.lua file on updating the doc
+-->
+
+To check current instances status use `status` command:
```
Comment (lenkis) [inline README.md:391]:
```suggestion
To check the current status of the instances, use the `status` command:
```

File: cartridge-cli.lua [hunk 4]
```diff
@@ -4050,23 +4055,51 @@ local function stop_all(args)
...
+
+        if warning ~= nil then
+            table.insert(warnings, string.format('%s: %s', process.instance_id, warning))
         end
 
         info('%s... %s', instance_status_str, res_str)
     end
 
+    if #warnings > 0 then
+        local warning = string.format('Some instances was not stopped:\n%s', table.concat(warnings, '\n'))
```
Comment (lenkis) [inline cartridge-cli.lua:4099]:
```suggestion
        local warning = string.format('Some instances were not stopped:\n%s', table.concat(warnings, '\n'))
```

File: cartridge-cli.lua [hunk 5]
```diff
@@ -3759,22 +3761,31 @@ local function start_all(args)
     local errors = {}
 
     -- start instances
-    for _, process_args in pairs(processes) do
-        local instance_status_str = string.format('Starting %s', process_args.instance_id)
+    for _, process in pairs(processes) do
```
Comment (olegrok) [inline cartridge-cli.lua:3764]: `pairs` doesn't respect of array. Is it important for us to give for user predictable result?

Comment (dokshina) [reply cartridge-cli.lua:3764]: Let's pretend you didn't see it

File: cartridge-cli.lua [hunk 6]
```diff
@@ -4109,23 +4114,51 @@ local function stop_all(args)
...
-
-        local ok, err = stop_process(process_args)
-        if not ok then
-            table.insert(errors, string.format('%s: %s', process_args.instance_id, err))
-            res_str = colored_msg('FAILED', ERROR_COLOR_CODE)
+        local err, warning
+
+        local status, status_err = process:get_status()
+
+        if status == nil then
```
Comment (olegrok) [inline cartridge-cli.lua:4127]: Looks I've seen similar piece of code here https://github.com/tarantool/cartridge-cli/pull/229/files#diff-65269a74ae0126ab3c284f41ebda16adR3771

Could you generalize it?

Comment (dokshina) [reply cartridge-cli.lua:4127]: There is an important difference between start and stop: `stop` interprets `Already stopped` as an warning, `start` interprets `Already started` as an error.

Moreover, I sure that generalizing starting and stopping will cause many errors. (for example, these command has different `already done` status)

File: test/integration/status_test.lua [hunk 7]
```diff
@@ -0,0 +1,238 @@
...
+    running_pids = {}
+end)
+
+g.after_each(function()
+    for _, pid in pairs(running_pids) do
+        helper.kill_process(pid)
+    end
+end)
+
+local function check_is_running(instance_fullname)
```
Comment (olegrok) [inline test/integration/status_test.lua:36]: Could you please move it into helpers (https://github.com/tarantool/cartridge-cli/blob/master/test/integration/start_stop_stateboard_only_test.lua#L36) ?

As it was done e.g. here - https://github.com/tarantool/cartridge/blob/master/test/helper.lua

**Pull request merged**